docpages/example_programs/the_basics/checking-member-permissions.md

```diff
@@ -1,11 +1,11 @@
 \page checking-member-permissions Checking permissions
 
 Of course most people do just iterate over the roles of a member to check for a permission.
-But there's a helper method for that: dpp::guild::base_permissions gets a member's permission taking into account the server owner and role permissions.
+But there's a helper method for that: `dpp::guild::base_permissions` gets a member's permission taking into account the server owner and role permissions.
 
-For total member permissions including channel overwrites use either the dpp::channel::get_user_permissions or dpp::guild::permission_overwrites method. Both do the same under the hood.
+For total member permissions including channel overwrites use either the `dpp::channel::get_user_permissions` or `dpp::guild::permission_overwrites` method. Both do the same under the hood.
 
-They all return a dpp::permission class, which is a wrapper around a permission bitmask containing bits of the dpp::permissions enum.
+They all return a `dpp::permission` class, which is a wrapper around a permission bitmask containing bits of the `dpp::permission`'s enum.
```

Comment:
No this is wrong, `dpp::permissions` is an enum, `dpp::permission` is a class, it was correct before